runtime/src/lib.rs

```diff
@@ -459,6 +461,12 @@ impl tee::Trait for Runtime {
     type OnReportWorks = Staking;
 }
 
+
+impl storage_order::Trait for Runtime {
+    type Event = Event;
+    type OnOrderStroage = (); // To do
```

Comment:
rename to `// TODO: Bonding with balance module`(this will be recognize in standard IDE), and add like new line above code due to the [house coding style](https://github.com/crustio/crust#rules)